SimTracker/SiPixelDigitizer/plugins/SiPixelDigitizerAlgorithm.cc

```diff
@@ -411,15 +414,16 @@ std::map<int, SiPixelDigitizerAlgorithm::CalParameters, std::less<int> > SiPixel
       // Testing the index conversion, can be skipped
       std::pair<int, int> p = PixelIndices::channelToPixelROC(chan);
       if (rowid != p.first)
-        cout << " wrong channel row " << rowid << " " << p.first << endl;
+        edm::LogInfo("PixelDigitizer ") << " wrong channel row " << rowid << " " << p.first << "\n";
       if (colid != p.second)
-        cout << " wrong channel col " << colid << " " << p.second << endl;
+        edm::LogInfo("PixelDigitizer ") << " wrong channel col " << colid << " " << p.second << "\n";
 
     }  // pixel loop in a ROC
 
-    cout << " map size  " << calmap.size() << " max " << calmap.max_size() << " " << calmap.empty() << endl;
+    edm::LogInfo("PixelDigitizer ") << " map size  " << calmap.size() << " max " << calmap.max_size() << " "
+                                    << calmap.empty() << "\n";
 
-    //     cout << " map size  " << calmap.size()  << endl;
+    //     edm::LogInfo("PixelDigitizer ") << " map size  " << calmap.size()  << "\n";
     //     map<int,CalParameters,std::less<int> >::iterator ix,it;
```

Comment:
@carolinecollard , we keep commented lines as an exception. AN alternative LogDebug instead of edm::LogInfo - this code will not be executed unless you start debugging. For debug purpose instead of uncommenting you may substitute LogDebug by edm::LogVerbatim